l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -181,19 +194,24 @@ def getCedPrest(self, cr, uid, cedPrest, context=None):
                     context=context
                 )
                 if not FiscalPosIds:
-                    raise orm.except_orm(
-                        _('Error!'),
+                    if context.get('inconsistencies'):
+                        context['inconsistencies'] += '\n'
+                    context['inconsistencies'] += (
                         _('RegimeFiscale type is not compatible ')
                     )
-                vals['register_fiscalpos'] = FiscalPosIds[0]
+                else:
+                    vals['register_fiscalpos'] = FiscalPosIds[0]
 
             if cedPrest.IscrizioneREA:
                 REA = cedPrest.IscrizioneREA
                 if not REA.NumeroREA:
```

Comment:
NumeroREA è un campo obbligatorio.
In questo caso o lasciamo il raise (eccezzione che non dovrebbe mai capitare , visto che la presenza di NumeroREA è già controllata da SDI) oppure lo togliamo proprio